man/docker.1.md

```diff
@@ -223,6 +223,70 @@ inside it)
   Block until a container stops, then print its exit code
   See **docker-wait(1)** for full documentation on the **wait** command.
 
+# CONFIGURATION FILES
```

Comment:
Hmm, do we really want this here? I think a more appropriate thing to do would be to create a man page for the `config.json` file and put this there.